### ISSUE TITLE
Chap 17 Bayesian Neural networks pt1

### ISSUE THREAD

**Issue by simonjdprince** [probml/pml2-book#20]
28th Feb version

P623 Eq 17.11 should probably define what \mathcal{S} is.  I guess some slightly non-typical notation for a softmax operation.
P624 L6 Missing reference.
P625 L18.  Split this sentence in two. You have a recursive structure of "which"s.

**Comment by murphyk** [comment]
fixed, thanks
